player/lua/stats.lua

```diff
@@ -995,3 +1137,22 @@ mp.register_event("video-reconfig",
             print_page(curr_page)
         end
     end)
+
+--  --script-opts-add=stats-help-keys=[-]{yes|<TERM-WIDTH>}
+if o["help-keys"] ~= "no" then
+    mp.command("no-osd set really-quiet yes")
```

Comment:
This can be shortened to

```suggestion
    mp.set_property("really-quiet", "yes")
```